### PR TITLE
Update mysql_4byte_support.rst

### PR DESCRIPTION
**Pull request by rth-dev**
could we place a note on how to verify whether the variable is set or not?

### CODE REVIEW

File: admin_manual/configuration_database/mysql_4byte_support.rst [hunk 1]
```diff
@@ -11,6 +11,15 @@ installation needs to be tweaked a bit.
 
         [mysqld]
         innodb_file_per_table=1
+        
+      Note:
```
Comment (MorrisJobke) [inline admin_manual/configuration_database/mysql_4byte_support.rst]: You need this to make it a code block:

```suggestion
      Note::
```

File: admin_manual/configuration_database/mysql_4byte_support.rst [hunk 2]
```diff
@@ -11,6 +11,15 @@ installation needs to be tweaked a bit.
 
         [mysqld]
         innodb_file_per_table=1
+        
+      Note::
```
Comment (MorrisJobke) [inline admin_manual/configuration_database/mysql_4byte_support.rst]:
```suggestion
      Note::
      
```

Comment (MorrisJobke) [reply admin_manual/configuration_database/mysql_4byte_support.rst]: It seems that a newline is also needed.

**Comment by rth-dev** [comment]
my very first time, just trying to figuring out how it works here 8)

**Pull request merged**